internal/tools/modformatter/modformatter.go

```diff
@@ -0,0 +1,219 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+
+// Package independent-lint checks a go.mod file at a given path specified by the -path argument
```

Comment:
```suggestion
// Package main checks a go.mod file at a given path specified by the -path argument
```
either you change this or line 9 I guess